gcloud-java-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryError.java

```diff
@@ -1,3 +1,18 @@
+/*
```

Comment:
You may want to include a basic checkstyle validator in compile or verify stage that would trigger a build failure. Similar that what was done [here](https://github.com/GoogleCloudPlatform/appengine-java-vm-runtime/blob/async-support/pom.xml#L217).

Such checks can include a license [header enforcer](http://checkstyle.sourceforge.net/config_header.html).